app/models/accounting/qb/purchase_extractor.rb

```diff
@@ -35,11 +35,11 @@ def set_type
 
       def extract_subtype
         payment_type = txn.quickbooks_data["payment_type"]
-        txn.qb_object_subtype = payment_type.to_s if payment_type.present?
+        txn.disbursement_type = (payment_type.present? && payment_type.to_s == "Check") ? :check : :other
```

Comment:
i will keep the presence check. the qb api is stricter than its UI on some things, and we might import something that doesn't have it - if for some reason it's missing we'd want this one to be other.

Reply:
. . . on second thought, we want a loud error if it's missing, since we'd be writing back the txn as a having a 'cash' payment type if we ever update it . . .